<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TwoCities_Path</class>
 <widget class="QWidget" name="TwoCities_Path">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mini Wasalni | Shortest Path Finder</string>
  </property>
  <widget class="QPushButton" name="traverse_button">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color: #F8F8F1;
		background-color: #A9724E;
		border: none;
		border-radius: 8px;
}

QPushButton:hover {
		color: #F8F8F1;
		background-color: #BFA17E;
		border: none;
		border-radius:8px;
}

QPushButton:pressed {
		color: #F8F8F1;
		background-color: #BFA17E;
		border: none;
        border-radius: 8px;
}</string>
   </property>
   <property name="text">
    <string>RETURN TO DASHBOARD</string>
   </property>
  </widget>
  <widget class="QLabel" name="shortestdistance">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>771</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: #909178;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="city1_label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: #909178;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="twopathfinder_h">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>581</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: #A9724E;
}</string>
   </property>
   <property name="text">
    <string>SHORTEST PATH FINDER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="city2_label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>430</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: #909178;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="distanse_between_label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: #F8F8F1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
